cfgov/jinja2/v1/_includes/organisms/filterable-list.html

```diff
@@ -0,0 +1,184 @@
+{# ==========================================================================
+
+   Filterable list
+
+   ==========================================================================
+
+   Description:
+
+   Create a filter list that includes controls and results when given:
+
+   value: Collection of field controls
+          and other settings to determine what's rendered.
+
+   value.no_posts_message: Message to show in "no posts" state.
+
+   value.no_posts_explanation: More info to show in "no posts" state.
+
+   value.form_type: What type of form this is.
+
+   value.categories.page_type: Page type of the filterable list control.
+
+   value.output_5050: Whether to render results as info-units or not.
+
+   value.link_image_and_heading: Whether to link the image and heading
+                                 in info-units.
+
+   filter_data.form: Django form that carries the fields to be rendered.
+
+   filter_data.page_set: Result set of posts from the filtered content.
+
+   ========================================================================== #}
+
+{% import 'organisms/filterable-list-controls.html' as filterable_list_controls with context %}
+{% import 'molecules/notification.html' as notification with context %}
+
+{# SHOW A NOTIFICATION IF THERE'S NOTHING TO FILTER. #}
+
+{% if value.no_posts_message %}
```

Comment:
This `if`/`elif` conditional needs to be in the opposite order. Currently, if I fill in the `no_posts_message` field on the "Blog" page, for example (even if there are posts), it acts like there are no posts:

<img width="800" alt="screen shot 2019-01-17 at 10 28 04 am" src="https://user-images.githubusercontent.com/4295388/51329029-99ae2000-1a42-11e9-806d-a04be18877f6.png">

Reply:
blergh, good catch. I pushed up a `has_results` variable and check that.

Reply:
Updated to `has_unfiltered_results` to better describe it since it had a value when the filters were applied and the warning notification was showing. https://github.com/cfpb/cfgov-refresh/pull/4771/commits/b322bf00c90571c5ca021608bd29adc5308fe4e2